### DISCUSSION TITLE
webkul market place not compatible with breeze theme

### DISCUSSION THREAD

**Discussion by arunguru77** [breezefront/community#121] [3. Integrations]
Hi support team,
we are looking a comatiable fix for webkul multvendor market place.
https://store.webkul.com/magento2-multi-vendor-marketplace.html
Please suggest.

Kind regards

**Comment by dimdimich** [comment]
Thank you for your interest. Webkull multi vendor modules is really complex solution and the best option would be to contact Webkul and ask to add Breeze support from their side.
